cmd/flags/admin.go

```diff
@@ -75,7 +75,7 @@ func (s *AdminServer) setLogger(logger *zap.Logger) {
 func (s *AdminServer) AddFlags(flagSet *flag.FlagSet) {
 	flagSet.Int(healthCheckHTTPPort, 0, healthCheckHTTPPortWarning+" see --"+adminHTTPHostPort)
 	flagSet.Int(adminHTTPPort, 0, adminHTTPPortWarning+" see --"+adminHTTPHostPort)
-	flagSet.String(adminHTTPHostPort, s.adminHostPort, "The host:port (e.g. 127.0.0.1:5555 or :5555) for the admin server, including health check, /metrics, etc.")
+	flagSet.String(adminHTTPHostPort, s.adminHostPort, "The host:port (e.g. 127.0.0.1:14269 or :14269) for the admin server, including health check, /metrics, etc.")
```

Comment:
This one might be tricky, as the admin port is used by multiple places with different values:

```console
$ podman run -e SPAN_STORAGE_TYPE=memory docker.io/jaegertracing/jaeger-ingester:latest --help | grep "admin.http.host-port string"
      --admin.http.host-port string                   The host:port (e.g. 127.0.0.1:5555 or :5555) for the admin server, including health check, /metrics, etc. (default ":14270")
```

Perhaps it should use `fmt.Sprintf("The host:port (e.g. 127.0.0.1:%s...", s.adminHostPort, ...)`

Reply:
Good catch! I've updated the code.